packages/typespec-ts/test/modularIntegration/generated/lro_modular/generated/src/api/operations.ts

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  StandardContext as Client,
+  isUnexpected,
+  CreateOrReplace200Response,
+  CreateOrReplace201Response,
+  CreateOrReplaceDefaultResponse,
+  DeleteOperation202Response,
+  DeleteOperationDefaultResponse,
+  Export202Response,
+  ExportDefaultResponse,
+} from "../rest/index.js";
+import { StreamableMethod } from "@azure-rest/core-client";
+import {
+  GetLongRunningPollerOptions,
+  getLongRunningPoller,
+} from "../common/lroImpl.js";
+import { SimplePollerLike, OperationState } from "@azure/core-lro";
+import { User, OperationStatus, ExportedUser } from "./models.js";
+import { RequestOptions } from "../common/interfaces.js";
+
+export interface CreateOrReplaceOptions extends RequestOptions {}
+
+export function _createOrReplaceSend(
+  context: Client,
+  role: string,
+  name: string,
+  options: CreateOrReplaceOptions = { requestOptions: {} }
+): StreamableMethod<
+  | CreateOrReplace200Response
+  | CreateOrReplace201Response
+  | CreateOrReplaceDefaultResponse
+> {
+  return context
+    .path("/azure/core/lro/standard/users/{name}", name)
+    .put({
+      allowInsecureConnection: options.requestOptions?.allowInsecureConnection,
+      skipUrlEncoding: options.requestOptions?.skipUrlEncoding,
+      headers: { ...options.requestOptions?.headers },
+      body: { role: role },
+    });
+}
+
+export async function _createOrReplaceDeserialize(
+  result:
+    | CreateOrReplace200Response
+    | CreateOrReplace201Response
+    | CreateOrReplaceDefaultResponse
+): Promise<User> {
+  if (isUnexpected(result)) {
+    throw result.body;
+  }
+
+  return {
+    name: result.body["name"],
+    role: result.body["role"],
+  };
+}
+
+/** Creates or replaces a User */
+export async function beginCreateOrReplace(
+  context: Client,
+  role: string,
+  name: string,
+  options: CreateOrReplaceOptions = { requestOptions: {} }
+): Promise<SimplePollerLike<OperationState<User>, User>> {
+  const pollerOptions = {
+    requestMethod: "PUT",
+    requestUrl: "/azure/core/lro/standard/users/{name}",
```

Comment:
not sure why we need this two options ? the send operation already contains such information.

Reply:
this is for LRO helper not for send func